substratevm/src/com.oracle.svm.reflect/src/com/oracle/svm/reflect/serialize/hosted/SerializationFeature.java

```diff
@@ -96,13 +121,140 @@ public void duringSetup(DuringSetupAccess a) {
                         ConfigurationFile.SERIALIZATION.getFileName());
     }
 
+    private static GraphBuilderConfiguration buildLambdaParserConfig(ClassInitializationPlugin cip) {
```

Comment:
Why do we need `cip`, it can be inlined.

Reply:
Fixed as suggested.